gms/feature_service.hh

```diff
@@ -113,6 +113,7 @@ public:
     gms::feature truncate_as_topology_operation { *this, "TRUNCATE_AS_TOPOLOGY_OPERATION"sv };
     gms::feature secondary_indexes_on_static_columns { *this, "SECONDARY_INDEXES_ON_STATIC_COLUMNS"sv };
     gms::feature tablets { *this, "TABLETS"sv };
+    // UUID_SSTABLE_IDENTIFIERS are always enabled
     gms::feature uuid_sstable_identifiers { *this, "UUID_SSTABLE_IDENTIFIERS"sv };
```

Comment:
It should be removed in the same patch, I don't know how the code reacts to having it both in a feature and in the grandfathered set.